src/Hl7.Fhir.Specification.Tests/Source/ZipSourceTests.cs

```diff
@@ -19,7 +20,8 @@ public class ZipSourceTests
         public void ListSummariesIncludingSubdirectories()
         {
             var zipfile = Path.Combine("TestData", "ResourcesInSubfolder.zip");
-            var zip = new ZipSource(zipfile, new DirectorySourceSettings() { IncludeSubDirectories = true });
+            var extractDir = Path.Combine(Path.GetTempPath(), Guid.NewGuid().ToString());
+            var zip = new ZipSource(zipfile, extractDir, new DirectorySourceSettings() { IncludeSubDirectories = true });
```

Comment:
Why should we pass here the extract directory? There is still a constructor of `ZipSource` that generates a default extraction directory. Right?